docs/lib/datastore/fragments/android/real-time/observe-snippet.md

```diff
@@ -9,3 +12,21 @@ Amplify.DataStore.observe(Post.class,
     () -> Log.i("GetStarted", "Observation complete.")
 );
 ```
+
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+Amplify.DataStore.observe(Post::class.java,
+    { Log.i("GetStarted", "Observation began.") },
```

Comment:
Can we change these tags to `MyAmplifyApp` across this pull? I'd like to my them uniform across the whole Android section.

Reply:
fixed